ops/transactional_phone_deploy.py

```
@@ -0,0 +1,390 @@
+
+
+class Abort(RuntimeError):
+    pass
+
+
+def emit(message: str) -> None:
+    print(message, flush=True)
+
+
+def run(argv: list[str], cwd: Path, *, check: bool = True, capture: bool = True) -> subprocess.CompletedProcess[str]:
+    result = subprocess.run(argv, cwd=cwd, text=True, capture_output=capture)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)